video/out/drm_common.c

```diff
@@ -814,6 +821,9 @@ bool vt_switcher_init(struct vt_switcher *s, struct mp_log *log)
     vt_mode.mode = VT_PROCESS;
     vt_mode.relsig = RELEASE_SIGNAL;
     vt_mode.acqsig = ACQUIRE_SIGNAL;
+#if HAVE_CONSIO_H
+    vt_mode.frsig = SIGIO; // unused but must be set
```

Comment:
What happens if we set `vt_mode.frsig` to `SIGIO` on all platforms, avoiding any need for ifdeffery at all? Why make this depend on `HAVE_CONSIO_H`? IIRC this isn't dependant on having consio but on being on BSD.

Reply:
Probably nothing. Neither Linux nor Solaris use the signal for forced release. Skipping may require documenting why `sigaddset` is not called.

Reply:
Looks like OpenBSD also checks if `frsig` is valid, see https://github.com/openbsd/src/commit/3832b9fd9698